.github/workflows/tests.yml

```
@@ -41,6 +41,9 @@ jobs:
     - name: Creating Zulip API key
       run: echo -e "${{ secrets.ZULIP_API_KEY }}" >> download
 
+    - name: Exporting Secret Key
+      run: export SECRET_KEY=${{ secrets.GITHUB_TOKEN }}
```

```suggestion
      run: echo -e "SECRET_KEY=${{ secrets.GITHUB_TOKEN }}" >> .env
```
Try with this @sahil9001.